test/e2e/cypress/e2e/clusters_overview.cy.js

```diff
@@ -165,4 +165,19 @@ context('Clusters Overview', () => {
       cy.contains(hanaCluster1.name).should('not.exist');
     });
   });
+
+  describe('Restoration', () => {
+    const hanaClusterToRestore = {
+      name: 'hana_cluster_1',
+      hosts: [
+        '13e8c25c-3180-5a9a-95c8-51ec38e50cfc',
+        '0a055c90-4cb6-54ce-ac9c-ae3fedaf40d4',
+      ],
+    };
+
+    it(`should show cluster ${hanaClusterToRestore.name}`, () => {
```

Comment:
I find this test too basic.
I think we agreed that we need to test if the previously added tags are restored properly